lintreview/github.py

```diff
@@ -26,12 +26,12 @@ def get_repository(config, user, repo):
     return gh.repository(owner=user, repository=repo)
 
 
-def get_lintrc(repo):
+def get_lintrc(repo, ref=None):
```

Comment:
Not at all, just didn't want to break anything if the function was used elsewhere, but it doesn't seem to be.